server/src/main/java/org/elasticsearch/persistent/PersistentTasksNodeService.java

```diff
@@ -197,9 +197,11 @@ public Task createTask(long id, String type, String action, TaskId parentTaskId,
         } finally {
             if (processed == false) {
                 // something went wrong - unregistering task
-                logger.warn("Persistent task [{}] with id [{}] and allocation id [{}] failed to create", task.getAction(),
-                        task.getPersistentTaskId(), task.getAllocationId());
+                ParameterizedMessage e = new ParameterizedMessage("Persistent task [{}] with id [{}] and allocation id " +
+                    "[{}] failed to create", task.getAction(), task.getPersistentTaskId(), task.getAllocationId());
+                logger.warn(e.getFormattedMessage());
```

Comment:
why did you change the logging here? AFAICS that change was unnecessary

Reply:
It's the same message, just reusing it for the logger and the exception so it's not c/p.  Can just write it out twice if you prefer.